sql/src/test/java/org/apache/druid/sql/calcite/CalciteJoinQueryTest.java

```diff
@@ -640,28 +660,28 @@ public void testFilterAndGroupByLookupUsingJoinOperatorAllowNulls(Map<String, Ob
 
     testQuery(
         "SELECT lookyloo.v, COUNT(*)\n"
-            + "FROM foo LEFT JOIN lookup.lookyloo ON foo.dim2 = lookyloo.k\n"
-            + "WHERE lookyloo.v <> 'xa' OR lookyloo.v IS NULL\n"
-            + "GROUP BY lookyloo.v",
+        + "FROM foo LEFT JOIN lookup.lookyloo ON foo.dim2 = lookyloo.k\n"
```

Comment:
There are lot of formatting changes hence its very easy to skip the actual change. 
* Couple of tests are marked notMsqCompatible();
* sortIfSortBased result validation.

Are there more changes in this class?

Reply:
Ah, yeah, that was the result of applying formatting in IntelliJ. I guess the last time this file was updated, the formatting wasn't applied. To help with reading the diff, Github has this "hide whitespace" feature for PR diffs. If you turn that on it looks like this:

https://github.com/apache/druid/pull/14105/files/ec47a89a785eaf5144aaf9db4752b29e002457c0?w=1#diff-4d1d101fd375322e3a765ae6aa738283f84967d508cf8afccf6f1572af2cd1c7

The substantive changes are indeed adding various `notMsqCompatible` and `sortIfSortBased`.